k8s/apis/federation/v1/catalog_eval_types.go

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Design question - why evaluation of catalog is implemented as a separate resource type?

Reply:
There are two reasons for that.
1. considering ARM scenario, we are not able to direct call symphony. All requests are via k8s bridge. We have to leverage bridge resource action to achieve this.
2. Symphony api are not reachable except symphony k8s controller and api itself. We have to do this via k8s controller.

Reply:
It feels strange to take this approach. Let's discuss in engineering meeting.